### ISSUE TITLE
How to start Serial Monitor without a COM Port selectable?

### ISSUE THREAD

**Issue by Zabz1** [mqnc/MHTinyCDC#2]
i am running your examples but i didn't get how i can open the Serial Monitor from the Arduino IDE without a selectable port.
Perhaps you're accessing it differently?

**Comment by mqnc** [comment]
No, I am using the serial monitor in the Arduino IDE, I see a port appearing. Maybe it's a driver or permission issue on your OS? If you're on Windows, maybe check this https://github.com/digistump/DigistumpArduino/issues/40

**Comment by Zabz1** [comment]
okay, i have removed previous drivers and installed the one you've linked but i still don't get a COM port.
When exactly should it be visible as one tho?
i am flashing over: 

```
Running Digispark Uploader...
Plug in device now... (will timeout in 60 seconds)
[...]
>> Micronucleus done. Thank you!
```

where i plug in the MH-ET Tiny88 after compile&upload.

Do i have to burn a bootloader or something to make the COM port appear, or how did you do it from blank?

EDIT:

it must be a driver issue as you suggested
![image](https://user-images.githubusercontent.com/95343277/195988767-d0906dd0-7116-4401-af09-3d42458c1c99.png)
what drivers do i need aditionally?

![image](https://user-images.githubusercontent.com/95343277/195989334-ff23b486-a755-4264-a569-90507140bb55.png)
im uploading via this board

**Comment by mqnc** [comment]
Sorry, I am using linux, so I cannot give super specific information. Looks like you flashed successfully. In theory, if you unplug and replug the device, it should appear as "Digispark Bootloader" in the device manager like in your picture for 5 seconds and then as a CDC device. In general, the trouble shooting should be the same as with a Digispark, so maybe google for "Digispark CDC Windows" or "DigiCDC Windows".

http://digistump.com/board/index.php/topic,2321.msg13831.html#msg13831

This looks pretty promising to me.

**Comment by Zabz1** [comment]
okay thank you.

I am stuck trying to use the drivers from LowCDC-Win10x64-1.0.1.6

after turning off the certificate verification in Windows startup settings
i did:

> 1- Install certificate using rigth-click on certcopy.cer file into "Local Machine" / "Trusted Root Certification Authorities" store.
Wait for confirmation ...
2- Install driver using rigth-click on lowcdc.inf file. Wait for confirmation ...

but when i try to select these drivers in hardware manager, i always get a Blue Screen regarding the lowcdc.sys file
im on Windows 11 and this seems to be the end of the road for me.

i guess i have to switch to Linux for this

**Comment by mqnc** [comment]
Ah, I wasn't even aware windows 11 already exists :D
I wanted to try on my windows 10 machine today but then that wouldn't help either.
It also somehow didn't work on my linux machine when I tried yesterday, will have to look into it...
Maybe WSL is an option for you? You can also run graphical applications with that, not just the terminal. But I don't know if it gets low level access to USB or if you still need working drivers in windows...

**Comment by Zabz1** [comment]
under the hood Windows 11 shares a lot of stuff with Windows 10 so chances are good findings from 10 can be applied on 11 as well. I will try the WSL route (i didnt know you can launch graphical stuff too) - thanks for the tip. will let you know if i was able to achieve something

**Comment by Zabz1** [comment]
Im running Arduino from WSL 2 (Ubuntu) now.
i am getting 
`.arduino15/packages/mhetlive/tools/micronucleus/2.0a4/micronucleus: error while loading shared libraries: libusb-0.1.so.4: cannot open shared object file: No such file or directory`

How did you install the drivers under Linux?

**Comment by mqnc** [comment]
Sorry I didn't get back to you, I had family business to attend. Now I also seem to have the same problem under Linux. If I use the "print" example, the chip keeps rebooting after the 5s, so it always appears as programmable. Did you make any progress?

**Comment by mqnc** [comment]
Did you try `sudo apt-get install libusb-0.1-4`?
